user_guide_src/source/installation/backward_compatibility_notes.rst

```diff
@@ -13,7 +13,8 @@ However, the code is not mature and bug fixes may break compatibility in minor r
 What are not Breaking Changes
 *****************************
 
-- The deprecated Config items are not covered by backwards compatibility (BC) promise. It may be removed in the next
-  **minor** version or later.
+- The deprecated items are not covered by backwards compatibility (BC) promise. It may be removed in the next next
```

Comment:
Duplicate words

```suggestion
- The deprecated items are not covered by backwards compatibility (BC) promise. It may be removed in the next
```

Reply:
No, it is not duplicate. Not the next, but next of the next. What should I put?
See the example after that.

Reply:
Maybe `in the second minor release`? However, I have no problem with understanding the current one.

Reply:
These are all fine. We're also all developers, so "in the `n+2` minor release" 🤣